2_process/calc_doy_means.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+import os
+
+def read_pred_csv(csv_path):
+    df = pd.read_csv(csv_path,
+                     parse_dates = ['date'],
+                     infer_datetime_format = True)
+    return df
+
+def calc_doy_means(df, site_id, out_file=None):
+    doy_means = df.groupby([df.date.dt.month, df.date.dt.day]).mean()
+    doy_means = doy_means.reset_index(drop=True)
+    doy_means.index.name = "doy"
+    doy_means["site_id"] = site_id
+    if out_file:
+        doy_means.to_csv(out_file)
+    return doy_means
+
+def main():
+    lake_ids = ["120020150", "107072210"]
+    for i in lake_ids:
+        pred_csv_file = os.path.join("out_data/tmp",
```

Comment:
Tweak to point to the right file path:
```suggestion
        pred_csv_file = os.path.join("1_fetch/out/tmp",
```